pkg/monitoring/metrics/virt-controller/vmistats_collector.go

```diff
@@ -75,6 +75,8 @@ var (
 			"guest_os_kernel_release", "guest_os_machine", "guest_os_arch", "guest_os_name", "guest_os_version_id",
 			// State info
 			"evictable", "outdated",
+			// Pod info
+			"vmi_pod",
```

Comment:
fixed

Reply:
@machadovilaca Shirly asked to get it back to vmi_pod since there is already the container pod label in this metric 
and it will overwrite the existing label? What if there is a race condition and in some cases the other value will take precedence?

WDYT?

Reply:
right... it is what it is, `vmi_pod` sounds fine then

Reply:
ok so fixed back